shared/ssz/ssz_utils_cache.go

```diff
@@ -110,3 +110,23 @@ func structFields(typ reflect.Type) (fields []field, err error) {
 	}
 	return fields, nil
 }
+
+func structFieldMap(typ reflect.Type) (fieldMap map[string]field, err error) {
```

Comment:
what does this have to do with `ProcessBlockHeader`?

Reply:
It's part of ssz.SigningRoot which is used in there